### PR TITLE
Update FTP.php (key param is not needed for sftp type)

### PR DESCRIPTION
**Pull request by dimkanio**
for symphony sftp doesnt work
  [PHPUnit\Framework\Exception] file_get_contents(): Filename cannot be empty  

so key is not needed in this case

### CODE REVIEW

File: src/Codeception/Module/FTP.php [hunk 1]
```diff
@@ -82,7 +82,19 @@
  *              key: ''
  *              tmp: 'tests/_data/ftp'
  *              cleanup: false
+ * *  *  * #### Example (SFTP) for symphony
```
Comment (Naktibalda) [inline src/Codeception/Module/FTP.php]: what is this symphony thing?

Comment (dimkanio) [reply src/Codeception/Module/FTP.php]: framework symfony

Comment (Naktibalda) [reply src/Codeception/Module/FTP.php]: how is FTP module related to symfony framework?

Comment (dimkanio) [reply src/Codeception/Module/FTP.php]: i use codeception with this framework

Comment (DavertMik) [reply src/Codeception/Module/FTP.php]: I agree, symfony or symphony are not related here :)

**Comment by Naktibalda** [comment]
The code change makes sense, it is just the description that didn't.

The problem this pull request fixes is that when the key field isn't set in suite configuration file,
it has the default empty string value and FTP module considers it as set and tries to load the file.
This issue can be fixed either by changing default value to `null` or by changing the code to consider empty strig as not set.

I will fix this issue in [module-ftp](https://github.com/Codeception/module-ftp)

**Comment by Naktibalda** [comment]
I raised new pull request to module-ftp: https://github.com/Codeception/module-ftp/pull/1

**Pull request closed without merging**